packages/react/src/components/ImageWithCaption/README.md

```diff
@@ -0,0 +1,76 @@
+# ImageWithCaption
```

Comment:
This appears to have snuck in, was this intended?

Reply:
oh whoops odd - removing it!